### PR TITLE
Delete deprecated AppTheme code

### PR DESCRIPTION
**Pull request by marlenecota**
#3701 

We need to keep AppTheme until we have an upstream solution for High Contrast, but we can get rid of the current theme code as that's now covered by the Appearance module. 

###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/microsoft/react-native-windows/pull/5883)

### CODE REVIEW

File: vnext/src/RNTester/js/examples-win/Theming/ThemingExample.windows.tsx [hunk 1]
```diff
@@ -6,23 +6,23 @@
 
 import React = require('react');
 import {Text, View, Button} from 'react-native';
-import {AppTheme} from '../../../../Libraries/AppTheme/AppTheme';
-import {IAppThemeChangedEvent} from '../../../../Libraries/AppTheme/AppThemeTypes';
+import {Appearance} from 'react-native';
```
Comment (NickGerleman) [inline vnext/src/RNTester/js/examples-win/Theming/ThemingExample.windows.tsx:9]: Appearance already has its own test page. Seems like we should just remove this one altogether.

File: vnext/src/Libraries/AppTheme/AppThemeTypes.ts [hunk 2]
```diff
@@ -18,12 +18,6 @@ export interface IHighContrastColors {
   WindowTextColor: string;
 }
 
-export type AppThemeTypes = 'light' | 'dark';
-
-export interface IAppThemeChangedEvent {
```
Comment (NickGerleman) [inline vnext/src/Libraries/AppTheme/AppThemeTypes.ts:23]: You'll probably need to run "yarn api" from the repo root after making TS API changes

**Comment by acoates-ms** [comment]
We shouldn't remove these APIs until we've released at least one stable release where using these APIs produces a yellowbox.

**Comment by NickGerleman** [comment]
Adding the Breaking Change label since this changes our public API and we'll want to document it in the release notes for 0.64.

**Comment by NickGerleman** [comment]
Hah, missed @acoates-ms response right before mine 🙂. Yellowboxing now and waiting until 0.65 to remove might be the better option. This one is obvious enough where I don't think it would be too damaging if we didn't, but it's definitely what we want to do in the future when we're planning deprecation.

**Comment by marlenecota** [comment]
#5883 will deprecate/yellowbox in 0.64. We will then delete the code in 0.65

**Comment by marlenecota** [comment]
Reviving PR as #5889 got backported to 0.63, so we can delete the code in 0.64

**Pull request merged**